src/Symfony/Component/Mailer/Bridge/Sendinblue/README.md

```diff
@@ -0,0 +1,12 @@
+Sendinblue Bridge
+=================
+
+Provides Sendinblue integration for Symfony Mailer.
+
```

Comment:
can you add like this, https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Mailer/Bridge/Mailjet/README.md,
a minimal config example? i think it is helpfull :)

Reply:
Having the format of the DSN is indeed a good addition.

Reply:
It's done. I've also added custom header example which is possible with Sendinblue.